crates/oxc_linter/src/snapshots/no_inner_declarations.snap

```diff
@@ -5,154 +5,154 @@ expression: no_inner_declarations
   ⚠ eslint(no-inner-declarations): Variable or `function` declarations are not allowed in nested blocks
    ╭─[no_inner_declarations.tsx:1:1]
  1 │ if (test) { function doSomething() { } }
-   ·             ──────────────────────────
```

Comment:
Is this intentional? Is this a better UX?

Reply:
Yes, otherwise you'll get a screenful of the function, which may span over dozen of lines. The diagnostic specifically talks about the function, not the function body.